src/es_archiver/lib/docs/generate_doc_records_stream.js

```diff
@@ -12,6 +12,18 @@ export function createGenerateDocRecordsStream(client, stats) {
         let remainingHits = null;
         let resp = null;
 
+        const alias = await client.indices.getAlias({ index });
+
+        if (!isEmpty(alias[index].aliases)) {
+          this.push({
```

Comment:
If it was me I would probably just make the aliases for an index a property here: https://github.com/elastic/kibana/blob/master/src/es_archiver/lib/indices/generate_index_records_stream.js#L26-L30, along with `index`, `settings`, and `mappings`.

Reply:
I agree that's where I thought it should go, too, but when I looked at it, I thought it would mess up the stream. I'll try moving it and see

Reply:
Ya. This is much better.